README.md

```diff
@@ -43,26 +43,26 @@ MapKnitter is broken into three major components:
 2. Application
 3. Exporting system
 
-**Component 1** has been broken out into a new Leaflet plugin, [Leaflet.DistortableImage](https://github.com/publiclab/Leaflet.DistortableImage/), which allows for client-side, CSS3-based distortion of images over a Leaflet base map
+**Component 1** has been broken out into a new Leaflet plugin, [Leaflet.DistortableImage](https://github.com/publiclab/Leaflet.DistortableImage/), which allows for client-side, CSS3-based distortion of images over a Leaflet base map.
 
 **Component 2** is a Ruby on Rails application which is the core of what you've looked at. It stores images, image corner locations, annotations, map details, and user accounts.
 
-**Component 3** is a set of calls to an external application, MapKnitter Exporter, which performs the distortions, geolocations, and produce export products like GeoTiff, TMS, jpg, etc. The external exporter service is built in a small Sinatra app at [mapknitter-exporter-sinatra](https://github.com/publiclab/mapknitter-exporter-sinatra) using the [mapknitter-exporter](https://github.com/publiclab/mapknitter-exporter) gem. Requests for exports are sent and progress is tracked using the API. 
+**Component 3** is a set of calls to an external application, MapKnitter Exporter, which performs the distortions and geolocations and produces export products like GeoTiff, TMS, JPG, etc. The external exporter service is built in a small Sinatra app at [mapknitter-exporter-sinatra](https://github.com/publiclab/mapknitter-exporter-sinatra) using the [mapknitter-exporter](https://github.com/publiclab/mapknitter-exporter) gem. Requests for exports are sent and progress is tracked using the API. 
 
 ## Installation
 
 Please consider which installation method you prefer. Cloud Installation requires fewer steps and is platform agnostic, but you may value working from your terminal, for familiarity, more.
 
-- [Cloud Installation](#Cloud-Installation)
-- [Standard Installation](#Standard-Installation)
-- [Windows Installation](#Windows-Installation)
+- [Cloud Installation](#cloud-installation)
+- [Standard Installation](#standard-installation)
+- [Windows Installation](#windows-installation)
 
 <hr>
 
 
 ## Cloud Installation
```

Comment:
This should be a subheading under `Installation`
```suggestion
### Cloud Installation
```

Reply:
Same goes for `Standard Installation`

Reply:
Okay. Thank you. 😄

Reply:
Since we're changing `Standard Installation` from h2 to h3, can I change the subheadings under it (like `Installation Steps` and `Prerequisites`) to a lower header level (h4) as they're currently at h3? Or is it okay as it is?

Reply:
@PeculiarE yeah you can change the subheading.